domains/zitado.json

```diff
@@ -0,0 +1,12 @@
+{
+        "owner": {
+           "username": "imadkm2006",
+           "email": "imadzm078@gmail.com",
+           "discord": "1191382601521451049"
+        },
+    
+        "record": {
+            "TXT": "google-site-verification=wxmynu5pblttmpktfsehl_uigvolgkm7s34iky2u878"
```

Comment:
Google site verification through TXT record is not supported, use the HTML method
also you need some contents on your site (unless it's for mails)